tests/jenesis/cmd/test_keys.py

```diff
@@ -0,0 +1,51 @@
+import subprocess
+import pytest
+import random
+import os
+from tempfile import mkdtemp
+import toml
+from jenesis.config import Config
+
+#@pytest.mark.skip
+def test_keys():
+    """Test key command"""
+
+    network = "fetchai-testnet"
+    profile = "profile_1"
+
+    path = mkdtemp(prefix="jenesis-", suffix="-tmpl")
+    os.chdir(path)
+
+    # create project
+    Config.create_project("project", profile, network)
+    os.chdir(os.path.join(path, "project"))
+
+    project_root = os.path.abspath(os.getcwd())
+
+    # read configuration file
+    input_file_name = "jenesis.toml"
+    toml_path = os.path.join(project_root, input_file_name)
+    with open(toml_path, encoding="utf-8") as toml_file:
+        data = toml.load(toml_file)
+
+    # set Jenesis to test keyring backend
+    data["project"]["keyring_backend"] = "test"
+
+    with open(toml_path, "w", encoding="utf-8") as toml_file:
+        toml.dump(data, toml_file)
```

Comment:
Looks like it might be useful to add a `keyring` argument when creating a project. We can keep this for now, though.

Reply:
Yeah that is a good idea